cmd/cmd.go

```diff
@@ -427,16 +428,33 @@ func generate(cmd *cobra.Command, model, prompt string, wordWrap bool, format st
 	var currentLineLength int
 	var wordBuffer string
 
-	request := api.GenerateRequest{Model: model, Prompt: prompt, Context: generateContext, Format: format}
-	fn := func(response api.GenerateResponse) error {
+	var role string
+	var fullResponse strings.Builder
+
+	request := api.GenerateRequest{Model: model, Prompt: prompt, Messages: messages, Format: format}
```

Comment:
Will this work for both completion and chat models?

Reply:
Yeah, messages will just build the prompt behind the scenes and pass it to the completion model. We will have to add some metadata to models to indicate they are completion models if we don't want to allow this. I can open a separate issue for that if you'd like.